hub/compression.py

```diff
@@ -25,9 +25,13 @@
 ]
 
 
+AUDIO_COMPRESSIONS = ["mp3"]
+
+
 BYTE_COMPRESSION = "byte"
 IMAGE_COMPRESSION = "image"
-COMPRESSION_TYPES = [BYTE_COMPRESSION, IMAGE_COMPRESSION]
+AUDIO_COMPRESSION = "audio"
```

Comment:
AUDIO_COMPRESSIONS and AUDIO_COMPRESSION? can we add a comment explaining the difference